app/jobs/split_pdf_job.rb

```diff
@@ -85,50 +102,51 @@ def save_pages(exam_template, partial_exams)
 
       group.access_repo do |repo|
         assignment_folder = exam_template.assignment.repository_folder
-        txn = repo.get_transaction(Admin.first.user_name)
-
+        unless Admin.first.nil?
```

Comment:
Don't use a block here. Do a one-liner, something like `return if Admin.empty?` (I believe `.empty?` or something like it can be used here too).